webdevcody/code-racer
# #303 Show submitting state when practice race is submitting

---
title: #303 Show submitting state when practice race is submitting
--=

Discord Username: @skwillzy

## What type of PR is this? (select all that apply)

- [*] 🍕 Feature

## Description

Added a RaceDetails component at the bottom of race that shows a submitting spinner and message when race details are being submitted. That component can also be used for future details such as a timer or other relevant details.

## Related Tickets & Documents

- Related Issue #303 
- Closes #

## QA Instructions, Screenshots, Recordings

Go to a practice race and while it's submitting and before it loads the results you will see the message and spinner

### UI accessibility concerns?

No accessibility concerns but it helps user feedback and experience.

## Added/updated tests?

- [*] 🙅 no, because they aren't needed

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?
